scout/server/blueprints/cases/templates/cases/utils.html

```diff
@@ -107,14 +107,27 @@ <h5 class="modal-title" id="exampleModalLabel">Beacon submission</h5>
 {% macro variant_transcripts(gene) %}
 <!-- Used in case_report template. Populate a table div with gene transcripts that are canonical, primary (max 5) or disease-associated -->
   {% set n_primary_txs = namespace(count=0) %}
+  {% set hgnc_primary = namespace(refseq=none, tx_id=none) %}
   <ul>
     {% for transcript in gene.transcripts %}
+
       <!-- increase counter if it has associated refseq -->
       {% if transcript.refseq_identifiers %}
         {% set n_primary_txs.count = n_primary_txs.count + 1 %}
       {% endif %}
 
-        <!-- show transcript info only if it's canonical, disease-associated or primary (print max 5 primary transcripts) -->
+      <!-- Capture HGNC primary transcript from primary transcript or refseq id-->
+      {% if hgnc_primary.refseq is none and transcript.is_primary %}
+        {% if transcript.transcript_id.startswith('NM_') %}
+          {% set hgnc_primary.refseq = transcript.transcript_id %}
```

Comment:
Yep, that was the idea! Fingers crossed it actually works!

Reply:
Oh right, did you check that the is_primary is actually set on the matching "other" refseq transcript as well? Lets see..

Reply:
Ok, well, it seems the HGNC primary transcripts are NOT versioned in their db. 🙄

Reply:
Nope!

Reply:
How about allowing multiple primaries then? That way we could flag both the ENS and NM_ one that both sort of match the HGNC primary?

Reply:
That is so silly of them!? Anyway, that also makes me feel that perhaps we are overdoing this. If they dont do this more carefully than yeah, well, that transcript whatever version, we probably also dont need to overdo it. We should just be very careful to write the *actual* transcript on the report, much like in the variant table.

Reply:
Yes but the versioned ones might be only the canonical, as in the example, and you would still miss them

Reply:
What about closing this PR and forgetting it ever existed? 😄

Reply:
Right, the issue is really that the full ID of a versioned transcript will not match the unversioned HGNC id.

Reply:
Yeah, I agree with closing. Your previous PR #2275 fixed the underlying issue we had. Then there are the two 1) ENSEMBL hg37 non-MANE mapping and 2) HGNC non-versioned. I don't have the heart to broach 1) with their devs again, and it is really inevitable that we switch to hg38 anyway. I think I will try pinging HGNC about their non-versioned transcripts. That is really bad form?!

Reply:
Perhaps they don't include the version in HGNC because it might change all the time and they would get complaints that stuff downloaded a month before is already old? I don't know otherwise

Reply:
I sent them a question - lets see!